web-library/src/main/java/ru/skypro/lessons/springboot/weblibrary/pojo/SecurityUserDetailsService.java

```diff
@@ -0,0 +1,26 @@
+package ru.skypro.lessons.springboot.weblibrary.pojo;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+
+public class SecurityUserDetailsService implements UserDetailsService {
```

Comment:
Вешаем сюда аннотацию `@Service`
```suggestion
@Service
public class SecurityUserDetailsService implements UserDetailsService {
```